lib/Access.php

```diff
@@ -1964,20 +1972,26 @@ private function initPagedSearch($filter, $bases, $attr, $limit, $offset) {
 					}
 				}
 				if ($cookie !== null) {
-					//since offset = 0, this is a new search. We abandon other searches that might be ongoing.
-					$this->abandonPagedSearch();
```

Comment:
There is an infinite loop in the "tests/integration/Lib/IntegrationTestPaging.php" caused by these changes. An unconditionally `abandonPagedSearch` fixes the problem.

Reply:
is there another condition that we can use to break the loop ?

Reply:
https://github.com/owncloud/user_ldap/pull/315/commits/9ed66eb3ecff1740cc622bcee70910510dd8024b should fix it, which solves part of the problems in https://github.com/owncloud/core/issues/6311